### PR TITLE
Dala 810 setup test server

### PR DESCRIPTION
**Pull request by fsoffel**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to a dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, execute npm run cypress and run Cypress Tests locally

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Please check all boxes before the Pull Request is merged. In case you skip a box
 - [ ] At least one E2E Test exists testing the new feature
 - [ ] Documentation is updated as required
 - [ ] The automated deployment is updated if required
-- [ ] The new version is deployed to a dev server using this branch
+- [ ] The new version is deployed to the dev server "dev1" using this branch
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply .github/pull_request_template.md:12]: Weil man in github kein oder festlegen kann, soweit ich gesehen habe. Es muss mindestens einmal nach dev1 deployed werden, damit gemerged werden kann.

File: prepareServerForDeployment.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply prepareServerForDeployment.sh:1]: Stimmt.

[the remark this answers is not in the document]
Comment (fsoffel) [reply prepareServerForDeployment.sh:1]: Done

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SiegfriedSobkowiak) [reply .github/pull_request_template.md:12]: ah ok

**Pull request merged**